scripts/start-eigenda-proxy.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+set -e
+
+if [ ! -e /data/verified ]; then
+	wget https://srs-mainnet.s3.amazonaws.com/kzg/g1.point --output-document=/data/g1.point
+	wget https://srs-mainnet.s3.amazonaws.com/kzg/g2.point.powerOf2 --output-document=/data/g2.point.powerOf2
+	wget https://raw.githubusercontent.com/Layr-Labs/eigenda-operator-setup/master/resources/srssha256sums.txt --output-document=/data/srssha256sums.txt
+  if (cd data && sha256sum -c srssha256sums.txt); then
+    echo "Checksums match. Verification successful."
+    touch /data/verified
+  else
+    echo "Error: Checksums do not match. Please delete this folder and try again."
+    exit 1
+  fi
+fi
+
+exec eigenda-proxy \
+  --addr=0.0.0.0 \
+  --port=4242 \
+  --eigenda-disperser-rpc=disperser-holesky.eigenda.xyz:443 \
+  --eigenda-eth-rpc=https://ethereum-holesky-rpc.publicnode.com \
```

Comment:
Should this be hardcoded, reuse `OP_NODE__RPC_ENDPOINT` or get it's own env var?